packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -810,8 +810,14 @@ export class VerticalStackedBarChartBase extends React.Component<
             />
           );
         }
-        if (barHeight < 1) {
+        let adjustedBarHeight = 0;
```

Comment:
This block of change looks redundant to me. We have already modified barHeight in previous lines.

Reply:
Removed redundant section in latest push